makefiles/docker.inc.mk

```diff
@@ -104,6 +104,10 @@ DOCKER_MAKE_ARGS ?=
 # https://github.com/docker/for-mac/issues/2396
 ETC_LOCALTIME = $(realpath /etc/localtime)
 
+# Fetch the number of jobs from the MAKEFLAGS
+DOCKER_MAKE_JOBS = $(filter -j%,$(MAKEFLAGS))
+
+DOCKER_MAKE_ARGS += $(DOCKER_MAKE_JOBS)
```

Comment:
AFAIK The number of jobs is only available after make 4.2, the one in docker is 4.1 no?

Reply:
That shouldn't cause a problem here I think. The number of jobs is filtered from the system `make`, not the docker container make. If the number of jobs is not in the `MAKEFLAGS` because the system installed `make` is too old, the result will be empty and no harm is done.

Reply:
Yeah but a comment on this would be nice

Reply:
> AFAIK The number of jobs is only available after make 4.2, the one in docker is 4.1 no?

Ahhh, now I get it. You're right. With Make older than 4.2, the *amount* of parallelism is not available in the `MAKEFLAGS`, only that parallelism is requested. So only `-j`, even if something like `-j5` is specified. That makes this a bit dangerous for users with make older than 4.2